ui/pages/swaps/prepare-swap-page/slippage-notification-modal.tsx

```diff
@@ -9,12 +9,12 @@ import {
 } from '../../../helpers/constants/design-system';
 import {
   Modal,
-  ModalContent,
-  ModalHeader,
   ModalOverlay,
   Box,
   ButtonPrimary,
 } from '../../../components/component-library';
+import { ModalContent } from '../../../components/component-library/modal-content/deprecated';
+import { ModalHeader } from '../../../components/component-library/modal-header/deprecated';
 import {
   SLIPPAGE_HIGH_ERROR,
   SLIPPAGE_LOW_ERROR,
```

Comment:
Screenshots showing no visual changes to UI

### Before
<img width="1512" alt="Screenshot 2023-12-10 at 5 08 42 PM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/bbfa5274-3269-4ca4-ab1b-0f9a18b4d3ad">

### After

<img width="1512" alt="Screenshot 2023-12-10 at 5 08 55 PM" src="https://github.com/MetaMask/metamask-extension/assets/8112138/ff350d33-340d-4486-aad0-49b1fb524a67">